ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/transactions/TransactionPoolTest.java

```diff
@@ -432,9 +450,19 @@ public void shouldAllowTransactionWhenAccountWhitelistControllerIsNotPresent() {
   public void shouldRejectRemoteTransactionsWhenNotInSync() {
     SyncState syncState = mock(SyncState.class);
     when(syncState.isInSync(anyLong())).thenReturn(false);
+    EthContext ethContext = mock(EthContext.class);
+    EthPeers ethPeers = mock(EthPeers.class);
```

Comment:
Why can't these mocks be created as fields and just reused? Would save a lot of noise through the tests.